web/src/main/java/se/inera/intyg/webcert/web/web/controller/integration/IntygIntegrationController.java

```diff
@@ -498,5 +512,22 @@ private void validateRequest(Map<String, Object> pathParameters) {
         super.validateParameters(pathParameters);
         super.validateAuthorities();
     }
+
+    private String assertLaunchIdFormat(String launchId) {
```

Comment:
The method does two things. Assert the launchId and also change and empty string to null. 

Also if launchId is null this would give a NullPointerException.